particles/Magenta/schemas/Sequence.arcs

```diff
@@ -0,0 +1,31 @@
+// Basic schema definitions for working with note sequences in magenta-js.
+// Distilled from review of https://hello-magenta.glitch.me/
+
+schema Note
```

Comment:
I think this is good for now, but the full `Note` spec can be found here:            https://github.com/tensorflow/magenta-js/blob/1bf6acc32b41a90987dfc908846cd2b3f0b69242/music/src/protobuf/proto.d.ts#L222